celery/utils/iso8601.py

```diff
@@ -1,7 +1,7 @@
 """Parse ISO8601 dates.
 
 Originally taken from :pypi:`pyiso8601`
-(http://code.google.com/p/pyiso8601/)
+(https://code.google.com/p/pyiso8601/)
```

Comment:
While we're at it can you update the link? https://bitbucket.org/micktwomey/pyiso8601